sdk/recoveryservices-siterecovery/Microsoft.Azure.Management.RecoveryServices.SiteRecovery/src/Microsoft.Azure.Management.RecoveryServices.SiteRecovery.csproj

```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.RecoveryServices.SiteRecovery</PackageId>
     <Description>Provides developers with libraries for the updated recovery services site recovery feature under Azure Resource manager.</Description>
-    <Version>2.2.0-preview</Version>
+    <Version>2.2.1-preview</Version>
```

Comment:
Track 1 is being deprecated, so dont need update this old sdk except you want to use this code for PowerShell cmdlets

Reply:
Reverted changes to "Track 1"